### ISSUE TITLE
Attributes are not initialized as an array if there are less than two

### ISSUE THREAD

**Issue by davidsantoso** [davidsantoso/markety#2]
When a new lead record is synced with Marketo, the response from Marketo is not converted into a new Markety Lead Record if the lead has one attribute or less. The issue is in the LeadRecord class, in self.from_hash()

``` ruby
def self.from_hash(savon_hash)
    lead_record = LeadRecord.new(savon_hash[:email], savon_hash[:id].to_i)

    savon_hash[:lead_attribute_list][:attribute].each do |attribute|
        lead_record.set_attribute(attribute[:attr_name], attribute[:attr_value], attribute[:attr_type])
    end

    lead_record
end
```

If there is only one attribute, savon_hash[:lead_attribute_list][:attribute] is just a hash of the single attribute and not an array of the attribute hashes which then causes a `TypeError: can't convert Symbol into Integer` error since .each() turns |attributes| into an array of the single attributes key/value pairs.

In other words, a lead records single attribute of 
`{:attr_name=>"FirstName", :attr_type=>"string", :attr_value=>"David"}` 
is iterated over as:
`[:attr_name, "FirstName"], [:attr_type, "string"], [:attr_value, "David"]`

When really, it needs to be:
`[{:attr_name=>"FirstName", :attr_type=>"string", :attr_value=>"David"}]` 
so that 
`lead_record.set_attribute(attribute[:attr_name], attribute[:attr_value], attribute[:attr_type])`
will work properly and set the attributes accordingly.
